### DISCUSSION TITLE
The lg-separator shouldn't have `role=separator`

### DISCUSSION THREAD

**Discussion by elenagarrone** [Legal-and-General/canopy#139] [General]
The `lg-separator` component has a `role="separator"` which in my opinion is going to be wrong in most cases as most of the time we'll use it just for styling. 

In my opinion the component should have an attribute of `aria-hidden="true"` by default and an input e.g. `hasRole` that allows a developer to have the role when needed.

**Comment by owensgit** [comment]
I agree, more often than not it's just a decoration. I think an input is a good idea for setting the role when needed 👍

**Reply by andy-polhill** [reply]
Should this become an issue in that case?

**Reply by elenagarrone** [reply]
I'll just update it now and create a PR.

**Comment by elenagarrone** [comment]
PR open #283
